integreat_cms/cms/utils/linkcheck_utils.py

```diff
@@ -243,7 +243,15 @@ def find_target_url_per_content(
     )
 
     links_to_replace = (
-        [link for link in links if link.url.type in link_types] if link_types else links
+        [
+            link
+            for link in links
+            if link.url.type in link_types
+            or link.url.status is False
+            and "invalid" in link_types
+        ]
```

Comment:
```suggestion
        (
            link
            for link in links
            if link.url.type in link_types
            or link.url.status is False
            and "invalid" in link_types
        )
```

Nit: This could be a generator to avoid creating an intermediate list which we don't need